src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -68,12 +74,18 @@ class PrettyErrorScreenListener
      * @param bool              $prettyErrorScreens True to render the error screens
      * @param \Twig_Environment $twig               The twig environment
      * @param ConfigAdapter     $config             The config adapter
+     * @param LoggerInterface   $logger             The logger service
      */
-    public function __construct($prettyErrorScreens, \Twig_Environment $twig, ConfigAdapter $config)
-    {
+    public function __construct(
+        $prettyErrorScreens,
+        \Twig_Environment $twig,
+        ConfigAdapter $config,
+        LoggerInterface $logger
```

Comment:
Well that's a matter of configuration, right? I could also change the templates… Just saying it looks like a convention to me, but I couldn't find any actually description/blog about it. Maybe someone else knows?